### ISSUE TITLE
Confirm outstanding a11y issues

### ISSUE THREAD

**Issue by gbinal** [18F/pulse#511]
I believe most/all of these were addressed earlier but we need to take the most recent feedback from Kathy and confirm.

**Comment by gbinal** [comment]
Here's the remaining feedback we received in the follow up report - 

![screen shot 2016-08-24 at 9 50 27 am](https://cloud.githubusercontent.com/assets/633088/17932984/554c2a48-69e0-11e6-9333-4090ec239ffb.png)

![screen shot 2016-08-24 at 9 50 39 am](https://cloud.githubusercontent.com/assets/633088/17932989/59773090-69e0-11e6-890b-faad68ef0b0a.png)

![screen shot 2016-08-24 at 9 50 47 am](https://cloud.githubusercontent.com/assets/633088/17932993/5ceaa4aa-69e0-11e6-9d0c-d0e7a37e96f5.png)

**Comment by gbinal** [comment]
We went through ^^^ with Dave and here's where we're at:   

1) We're in agreement that this is not a problem.  
2) This appears to actually be a misidentified issue.  What's actually at issue here is that some of the domains were missing their agency field (it can be seen by going [here](https://staging.pulse.cio.gov/accessibility/domains/) and sorting by `Agency`).  I believe this issue was a one time problem and is resolved in the new scan data we're using, but am tracking the issue at https://github.com/18F/pulse/issues/521.  
3) I believe this and 4 were addressed by #480 and are no longer outstanding.  This may be referring to the text that is being used for the tables and suggesting that the language be updated.  If so, that shouldn't be a blocker and can be something we discuss.  
4) ^^^^
5) There is a tension here in how we hyperlink on the site.  Linking to the issue tracker from `in the open` in the footer is good form in a number of ways.  Changing the text that is hyperlinked would negatively impact the  We're in agreement that this is a minor issue and not a blocker for release.  
6) This issue is suggesting more descriptive page titles throughout the site.  We may disagree on what is best here, namely how verbose page titles should be.  If desired though, we can update the page titles on the site very easily.  It should be done consistently across the site.  In the meantime, this is not a blocker for release.  
7) This is referring to the plus sign inside of the blue button.  We're in agreement that this is a minor issue and not a blocker for release.  Let's factor in with #191 and #392.

**Comment by gbinal** [comment]
@nickbristow - can you confirm my understanding of 3) and 4) above?

**Comment by nickbristow** [comment]
@gbinal 3 is addressed, 4 is not.

**Comment by gbinal** [comment]
Thanks.  I'm updating the analysis to that effect and filing an issue for 4.

**Comment by davidstenger** [comment]
Using JAWS, I do not see the issues that Kathy has found. The labels on the tabs are descriptive and provide the JAWS access key. The number of search items says show the number and then entries and then the numbers in the right corner speak the word page followed by the page number. For the tabs an ARIA Role is used, that may not be discussed in the Baseline. ARIA labels are used for the other areas. So, in my opinion, it seems fine.

**Comment by gbinal** [comment]
@davidstenger - Do you mean for the issue in 3) above?

**Comment by gbinal** [comment]
_From @davidstenger, a good color contrast change can be: `Replace #3399FF with #003366`._

**Comment by gbinal** [comment]
As I understand it, we're just doing 7) on this at this stage and it is addressed by https://github.com/18F/pulse/issues/551, so I'm planning to close this issue if possible.

**Comment by gbinal** [comment]
Closing.
